hudi-cli/src/main/java/org/apache/hudi/cli/commands/TempViewCommand.java

```diff
@@ -20,36 +20,55 @@
 
 import org.apache.hudi.cli.HoodieCLI;
 
+import org.apache.hudi.exception.HoodieException;
 import org.springframework.shell.core.CommandMarker;
 import org.springframework.shell.core.annotation.CliCommand;
 import org.springframework.shell.core.annotation.CliOption;
 import org.springframework.stereotype.Component;
 
-import java.io.IOException;
-
 /**
  * CLI command to query/delete temp views.
  */
 @Component
 public class TempViewCommand implements CommandMarker {
 
-  private static final String EMPTY_STRING = "";
+  public static final String QUERY_SUCCESS = "Query ran successfully!";
+  public static final String QUERY_FAIL = "Query ran failed!";
+  public static final String SHOW_SUCCESS = "Show all views name successfully!";
 
-  @CliCommand(value = "temp_query", help = "query against created temp view")
+  @CliCommand(value = {"temp_query", "temp query"}, help = "query against created temp view")
   public String query(
-          @CliOption(key = {"sql"}, mandatory = true, help = "select query to run against view") final String sql)
-          throws IOException {
+          @CliOption(key = {"sql"}, mandatory = true, help = "select query to run against view") final String sql) {
+
+    try {
+      HoodieCLI.getTempViewProvider().runQuery(sql);
+      return QUERY_SUCCESS;
+    } catch (HoodieException ex) {
+      return QUERY_FAIL;
```

Comment:
Can we log the detailed exception information before returning the result?

Reply:
`runQuery` had log the detailed exception.

Reply:
OK~